### ISSUE TITLE
Right sidebar has initial focus when open upon page visit

### ISSUE THREAD

**Issue by kulmann** [owncloud/web#10618]
## Describe the bug
We now persist the open/closed state of the right sidebar in the localStorage of the browser and restore that state on the next visit (has been implemented in https://github.com/owncloud/web/pull/10612 ).
In that case (open sidebar upon first page load) the right sidebar has the initial focus. That's an accessibility violation.

## Steps to reproduce
1. Open right sidebar anywhere
2. Close the browser
3. Reopen the browser

## Expected behavior
There is no initial focus set. First TAB always jumps to the hidden button which let's the user navigate to the `<main>` content.

## Actual behavior
Right sidebar is initially focussed. First TAB jumps to the `X` button to close the right sidebar.
